about/index.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+<html manifest="file.manifest">
+	
+    <head>
+        <meta charset="UTF-8">
+        <link rel="icon" type = "image/png" href="../img/favicon.png">
+        <link rel="apple-touch-icon" href="../img/favicon.png">
+        <meta name="description" content="The official website of the IEEE Goa College of Engineering Student Branch">
+        <meta name="apple-mobile-web-app-title" content="IEEE GEC SB Website">
+        <meta name="keywords" content="ieee, gec, goa engineering college , goa college of engineering, ieee gec,">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>IEEE GEC SB | About</title>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.css">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous"> <!-- Fontawesome -->
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css" integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB" crossorigin="anonymous">
+        <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous" defer></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous" defer></script>
+        <script src="script.js" type="text/javascript" async></script>
+        <script src="scrollreveal/dist/scrollreveal.min.js" defer></script>
+    </head>
+
+    <body>
+
+        <!-- Back to top button-->
```

Comment:
Unless you plan on animating the scroll, using an anchor would be a better approach.